lib/response.js

```diff
@@ -958,9 +959,24 @@ res.render = function render(view, options, callback) {
   };
 
   // render
-  app.render(view, opts, done);
+  app.render(view, varStack, done);
 };
 
+/**
+ * Add `options` to the top of the reponse's template variables stack.
+ * Creates the stack the first time called.
+ *
+ * @public
+ */
+res.addTmplVars = function addTmplVars(options) {
```

Comment:
We'll probably want to think of a better method name than `addTmplVars`, but I don't have anything on the top of my tongue. Just adding that I don't particularly care for this name, and would love to hear thoughts from other people on it :)